flang/runtime/command.cpp

```diff
@@ -125,6 +142,36 @@ static bool FitsInDescriptor(
       kind, terminator, value);
 }
 
+void removeNewLine(char *str) {
+  char *newlinePos = std::strchr(str, '\n');
+  if (newlinePos) {
+    *newlinePos = '\0'; // Replace with null terminator
+  }
+}
+
+std::int32_t RTNAME(FDate)(const Descriptor *value, const Descriptor *errmsg) {
+  FillWithSpaces(*value);
```

Comment:
Yes indeed, not a error but I believe something wrong with the char

```
If input char length is >26

Without FillWithSpaces
 Fri Nov 10 15:45:25 202  �
 
With FillWithSpaces
 Fri Nov 10 15:50:53 2023     
```

Reply:
never mind, it won't be a problem when I move in to `extension.cpp`. (have it implemented there, instead of calling a runtime function). and its probably the place where it should be. It is a `GNU extension` rather than a `standard extension`.
https://gcc.gnu.org/onlinedocs/gfortran/FDATE.html